PeerioTechnologies/peerio-icebear
# fix: unique id for getAuthToken [ch13658]

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/13658/sdk-retrying-auth-token-after-failed-login-attempt-always-fails

#### Testing instructions  

Login to staging with some user. Switch server. Reload the client. Signing up should work fine.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
